modules/exploits/windows/persistence/registry.rb

```diff
@@ -80,106 +85,70 @@ def generate_blob_reg
   end
 
   def generate_cmd_reg
-    cmd_reg = datastore['RUN_NAME'] || Rex::Text.rand_text_alphanumeric(8)
-    return cmd_reg
+    datastore['RUN_NAME'] || Rex::Text.rand_text_alphanumeric(8)
   end
 
   def install_blob(root_path, blob, blob_reg_key, blob_reg_name)
     blob_reg_key = "#{root_path}\\#{blob_reg_key}"
     new_key = false
-    if not registry_enumkeys(blob_reg_key)
+    if !registry_enumkeys(blob_reg_key)
       unless registry_createkey(blob_reg_key)
         fail_with(Failure::Unknown, "Failed to create key #{blob_reg_key}")
       end
       print_good("Created registry key #{blob_reg_key}")
       new_key = true
     end
 
-    unless registry_setvaldata(blob_reg_key, blob_reg_name, blob, "REG_SZ")
+    unless registry_setvaldata(blob_reg_key, blob_reg_name, blob, 'REG_SZ')
       fail_with(Failure::Unknown, 'Failed to open the registry key for writing')
     end
     print_good("Installed payload blob to #{blob_reg_key}\\#{blob_reg_name}")
     return new_key
   end
 
+  def runkey
```

Comment:
It is odd this does not match the datastore option. Would it be better to change the method to match the datastore value?  I think the preference here might be to change the method to `regkey`?